### ISSUE TITLE
Mod "Stuff of Building" duplicate crops stuff

### ISSUE THREAD

**Issue by DV8ZXPru** [TeamAOF/All-of-Fabric-5#33]
Just put the seeds in your inventory and right-click with the building stick in your hand over the crops. profit
![bg1](https://user-images.githubusercontent.com/2900356/146987570-cbbe0e77-b36f-427f-b027-81fa7fb5189e.png)
![bg2](https://user-images.githubusercontent.com/2900356/146987613-dbfaebda-0b27-4c5c-9c13-57d5da5ce7a8.png)

**Comment by acorryn** [comment]
To add to this, a similar duplication also happens with Turtle Eggs, Candles and Sea Pickles due to how the wand makes an exact copy of the target blocks' block states.

**Comment by Draylar** [comment]
Done & done - you can find the [new release here](https://www.curseforge.com/minecraft/mc-mods/staff-of-building/files/3577332) once it is approved. This fix should cover all crops & the misc blocks mentioned by acorryn. If you encounter any more problematic blocks, feel free to let me know here or open a new issue [over at the Staff of Building bug tracker](https://github.com/Draylar/staff-of-building). Thank you both for the reports!

**Comment by Monster-Zer0** [comment]
thank you @Draylar
